examples/example_vulkan.cpp

```diff
@@ -45,9 +47,177 @@ void testNonEngineIntergratedVulkan()
 
     std::vector<VkPhysicalDevice> physicalDevices = GetPhysicalDevices(vulkanInstance);
 
+    VkPhysicalDevice selectedDevice = VK_NULL_HANDLE;
+    std::vector<QueueInfo> queueInfos; // TODO: should be a set?
+
+    // TODO: Only temp.. These are all the extensions available on Dell XPS with GTX 1050.. using a subset of them as desiredExtensions now
+    const std::vector<char const *> desiredDeviceExtensions{
```

Comment:
Are you supposed to know about what extensions your device has?

Reply:
Yup.. you're supposed to provide the extensions you'll need. But this we can hide in DireWolf